src/android/build.gradle

```diff
@@ -34,5 +34,6 @@ android {
 
 dependencies {
     compile 'com.facebook.react:react-native:+'
+    compile 'com.squareup.okhttp3:okhttp:3.4.1'
```

Comment:
There's a [React Native PR to upgrade okhttp](https://github.com/facebook/react-native/pull/11835#discussion_r114119783) to 3.6.0 (or possibly 3.7.0). If and when that one is merged in, the version should change here as well.